packages/itwin/measure-tools/package.json

```diff
@@ -44,41 +44,44 @@
     "url": "http://www.bentley.com"
   },
   "peerDependencies": {
-    "@itwin/appui-abstract": "^4.7.0",
+    "@itwin/appui-abstract": "^4.7.0 || ^5.0.0",
```

Comment:
should we just drop support for 4?

Reply:
okay, dropped

Reply:
should prob update the appui deps to match the ver that supports core

Reply:
I've updated the peer deps for appui to be 5.6.0, also dev deps too.